src/main/java/no/nav/fpsak/tidsserie/StandardCombinators.java

```diff
@@ -78,6 +78,14 @@ public static <T, V> LocalDateSegment<List> bothValues(LocalDateInterval dateInt
         }
     }
 
+    /** Basic combinator som returnerer første (Left-Hand Side) verdi hvis begge finnes og er like. */
+    public static <V> LocalDateSegment<V> leftIfEqualsRight(LocalDateInterval dateInterval,
```

Comment:
Jeg mistenker at bruksmønsteret for denne kombinatoren er såpass smalt at den ikke bør inn i fp-tidsserie, og heller defineres der hvor den brukes

Reply:
og som du sier Jens Otto, så er den ikke en standard-kombinator, men heller et join on-ledd, så passer ikke 100% inn her